enterprise/models.py

```diff
@@ -2504,7 +2504,6 @@ class EnterpriseCustomerReportingConfiguration(TimeStampedModel):
     )
 
     DATA_TYPE_PROGRESS = 'progress'  # Refers to gathering progress data from Vertica (to be deprecated)
```

Comment:
Shouldn't we also need to remove `progress` type?

Reply:
done sir